core/src/main/java/org/frankframework/ldap/LdapClient.java

```diff
@@ -353,8 +352,7 @@ public Set<String> searchRecursivelyViaAttributes(String uid, String baseDn, Str
 			    			if (log.isDebugEnabled()) log.debug("nestingLevel ["+nestingLevel+"] found secondary membership ["+secondaryResult+"]");
 			    			results.add(secondaryResult);
 			    			if (!searched.contains(secondaryResult) &&
-			    				!searchingNow.contains(secondaryResult) &&
-			    				!toBeSearched.contains(secondaryResult)) {
```

Comment:
Deze is nu ook weg. Is toBeSearched een set?

Reply:
Ja, een `LinkedHashSet`.